CHANGELOG.md

```
@@ -3,25 +3,47 @@ All notable changes to this project will be documented in this file.
+- Added method to capture the actual number of pages. 
+
+---
+
+## [v7.0.0] - 2026-04-13
+### Added
+- Added PDF content search support using extracted document text.
+- Added OCR processing state tracking for file documents.
+- Added asynchronous OCR processing through a background worker and Cloud Run Job execution mode.
+- Added production OCR deployment configuration for Cloud Run and Cloud Scheduler integration support.
+
+## Changed
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)